addons/website_payment/static/src/snippets/s_donation/donation_snippet.js

```diff
@@ -36,7 +36,7 @@ export class DonationSnippet extends Interaction {
     }
 
     async willStart() {
-        this.currency = await rpc("/website/get_current_currency", { cached: true });
+        this.currency = await rpc("/website/get_current_currency", { cache: true });
```

Comment:
Hello @jpp-odoo @aab-odoo 

How is this cached in the frontend? If I understood the code well, the cache is initialised when we start the web client in the backend, but we don't use webClient in the frontend, right?

Reply:
We had to develop our own solution for a snippet: https://github.com/odoo/odoo/pull/213234/files#diff-9abc2f361adc53cc4b0c7923c26042b2e995ef40fc4926bc586c31aa2418ac58R26-R38 but I guess it will happen more often on eCommerce. Do you plan to add rpc cache to the frontend?